pom.xml

```diff
@@ -67,7 +67,7 @@
 		<slf4j-api.version>1.7.25</slf4j-api.version>
 
 		<bouncycastle.version>1.60</bouncycastle.version>
-		<junixsocket.version>2.0.4</junixsocket.version>
+		<junixsocket.version>2.1.2</junixsocket.version>
```

Comment:
```suggestion
		<junixsocket.version>2.2.0</junixsocket.version>
```

Reply:
Let's see if this gets us farther.  Also see https://github.com/kohlschutter/junixsocket/issues/31#issuecomment-464527380 for more suggestions.

Reply:
Yes, please directly update to 2.2.0. There's no point in moving anymore.